content/en/docs/concepts/extend-kubernetes/api-extension/custom-resources.md

```diff
@@ -105,7 +105,7 @@ Use a [secret](/docs/concepts/configuration/secret/) for sensitive data, which i
 Use a custom resource (CRD or Aggregated API) if most of the following apply:
 
 * You want to use Kubernetes client libraries and CLIs to create and update the new resource.
-* You want top-level support from kubectl (for example: `kubectl get my-object object-name`).
+* You want top-level support from `kubectl` (e.g. `kubectl get my-object object-name`).
```

Comment:
Oh, I didn't spot this before.
[Avoid Latin phrases](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-latin-phrases) and abbreviations if you can.

Reply:
Oh! Okay... I'll revert that.